tests/atlasgen/test_annotation_utils.py

```diff
@@ -12,27 +13,67 @@
 
 
 @pytest.mark.parametrize(
-    "input_name, expected_name, expected_acronym",
+    "input_name, expected_name, expected_acronym, acronym_length",
     [
         pytest.param(
-            "BrainGlobeAtlas Name (BGA-N)",
-            "BrainGlobeAtlas Name",
+            "BrainGlobeAtlas-Name (BGA-N)",
+            "BrainGlobeAtlas-Name",
             "BGA-N",
+            np.random.randint(1),
```

Comment:
I am confused why you are using `randint` here?
I would keep it simple and use 1, 2, len+1 for the three tests.